src/style/ParagraphProperties.ts

```diff
@@ -127,6 +139,20 @@ export class ParagraphProperties implements IParagraphProperties {
     }
   }
 
+  /**
+   * Sets the keep together attribute if paragraph marked as keep together.
+   *
+   * @param {Element} textPropertiesElement The element which will take the attribute
```

Comment:
Please fix the name of the attribute (the attribute actually is named `paragraphPropertiesElement`).

I just discovered it's wrong for the other methods in this class as well. If you don't mind, you may adjust the other JSDocs as well.

Reply:
Addressed in 3326a03e753f8b06119111c104538b78e87dde14